### PR TITLE
Display a sidebar in documentation page

### PR DESCRIPTION
**Pull request by JesusValeraDev**
# 📚  Description

Add a lateral menu on `Documentation` section.

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]

File: sass/_header.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
   top: 0;
   left: 0;
   right: 0;
-  height: $header-height;
+  //height: $header-height;
```
Comment (JesusValeraDev) [inline sass/_header.scss:8]: .

File: static/dark-mode.js [hunk 2]
```diff
@@ -3,23 +3,23 @@ document.addEventListener("DOMContentLoaded", _ => {
...
-            g.setAttribute('stroke', svgColor);
-        }
-    });
-
-    let gitHubLogo = document.getElementById('fill-github');
-    if (gitHubLogo !== null) {
-        gitHubLogo.setAttribute('fill', svgColor);
-    }
-    updateTheme();
+    // // let currentTheme = localStorage.theme !== 'dark' ? 'light' : 'dark';
```
Comment (JesusValeraDev) [inline static/dark-mode.js:6]: Let's keep this commented code for now 👍🏼

File: sass/_header.scss (hunk 1 above)
Comment (Chemaclass) [reply sass/_header.scss:8]: you can take care of this in the follow up PR when introducing back the dark-mode

File: sass/_base.scss [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+  color: $color-base-primary;
+  text-decoration: none;
+
+  &:hover,
+  &:focus { text-decoration: underline; }
+
+  &:hover { outline-width: 0; }
+}
+
+//* + * {
```
Comment (JesusValeraDev) [inline sass/_base.scss:34]: .

File: sass/_base.scss [hunk 4]
```diff
@@ -12,7 +71,8 @@ body {
 }
 
 .container {
-  width: $body-width;
+  //width: $body-width;
```
Comment (JesusValeraDev) [inline sass/_base.scss:74]: .

**Pull request merged**